### PR TITLE
main image adding

### PR DESCRIPTION
**Pull request by Dabrytskaya**
I`m just adding the main screen. I have absolutely no peace of mind for the code((

### CODE REVIEW

**Comment by yildiztugba** [comment]
@Dabrytskaya can you send again, there are lots of conflicts, i couldnt follow. (update your files & push)

**Comment by Dabrytskaya** [comment]
@yildiztugba look please, i closed the conflicts

File: css/style.css [hunk 1]
```diff
@@ -28,576 +28,585 @@ h2,
...
-    text-align: left;
-    max-width: 470px;
-    line-height: 160%;
-    font-size: 16px;
-    font-family: "Space Mono", monospace;
-    margin: 0px;
-    word-break: break-word;
-    font-weight: 300;
-    margin: 0px 0px 20px 0px;
+  padding: 0;
```
Comment (yildiztugba) [inline css/style.css:159]: This part is hasan's part. And i think (red-old one) program - info has many codes than (Green) New program-info. İf i merge this, do we lose the codes or not?

**Review by yildiztugba — CHANGES_REQUESTED** [review]

**Comment by yildiztugba** [comment]
A new pr will be sended by katia

**Pull request closed without merging**